catkin_tools/jobs/utils.py

```diff
@@ -25,6 +25,21 @@
 from catkin_tools.execution.events import ExecutionEvent
 
 
+class CommandMissing(Exception):
+    '''A required command is missing.'''
+
+    def __init__(self, name):
+        super(CommandMissing, self).__init__(
+                'Cannot find required tool `%s` on the PATH, is it installed?' % name)
```

Comment:
Happy to move this to its own `exceptions.py` file, didn't want to bother with that unless there was a clear need.